packages/material-ui-csskit/src/responsiveTypography.js

```diff
@@ -0,0 +1,88 @@
+import convertLength from 'convert-css-length';
+import responsiveProperty from './responsiveProperty';
+import alignProperty from './alignProperty';
+import fontGrid from './fontGrid';
+
+function isUnitless(value) {
+  return String(parseFloat(value)).length === String(value).length;
+}
+
+/**
+ *
+ * responsiveTypography params
+ * @typedef {Object} responsiveParams
+ * @property {number} maxScale - The scale that applies to the highest breapoint
```

Comment:
Bootstrap is handling the problem in the opposite direction:
- non-responsive typography: https://getbootstrap.com/docs/4.3/content/typography/
- responsive typography: https://project-rfs.github.io/docs/4.3/content/typography/

They start from the desktop values and reduce the font size for smaller screen devices. However, the value can't go below 16px: the viz: https://github.com/twbs/rfs#visualisation.

What should we do?

Reply:
not forgotten/missed ^^ just need more time to review their code